app/frontend/src/components/forms/submission/StatusPanel.vue

```diff
@@ -27,20 +27,76 @@
               @change="statusFields = true"
             />
 
+            <div v-show="statusFields">
```

Comment:
![image](https://user-images.githubusercontent.com/58443522/140555171-afe95439-2757-4b17-9106-2814e180380b.png)
These might be UI decisions for another time but i wonder if we should change this instruction. Its not that obvious what changing status to 'revising' does. Maybe it could say 'workflow action' or maybe it needs a tooltip